de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -414,28 +417,37 @@ private void triggerProjectNegotiation() {
             return;
 
         triggered = true;
+        IReferencePointManager referencePointManager = sessionManager.getSession().
+            getComponent(IReferencePointManager.class);
+        Map<String, IReferencePoint> localReferencePoints = new HashMap<>();
+
+        for(String key : localProjects.keySet()){
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Iterate over the "entrySet" instead of the "keySet". ([squid:S2864](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2864))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMgvDZywvPe4LUfj&open=AWZfKMgvDZywvPe4LUfj)

[ISSUE_KEY]: AWZfKMgvDZywvPe4LUfj